src/ui/Startup.cs

```diff
@@ -236,8 +236,6 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 };
             });
-            services.AddScoped<SearchAndCompare.UI.Shared.Features.IFeatureFlags, SearchAndCompare.UI.Shared.Features.FeatureFlags>();
```

Comment:
This one is still in use

Reply:
@timabell I don't see how it can be... `IFeatureFlags` has been deleted 🤔

Reply:
ah, but this is the *other* `IFeatureFlags` from the nuget package from search. The preview uses it

Reply:
Don't delete that, bad things will happen